gulp/FabricBuild.js

```diff
@@ -7,6 +7,29 @@ var ConsoleHelper = require('./modules/ConsoleHelper');
 var ErrorHandling = require('./modules/ErrorHandling');
 var Plugins = require('./modules/Plugins');
 
+var srcPath;
+var cssPlugin;
+var fileExtension;
+var prefixLinter;
+
+//
+// Configure Tasks
+// ----------------------------------------------------------------------------
+gulp.task('Fabric-configureBuild', function () {
+    // Check if building SASS
+    if(Config.buildSass) {
```

Comment:
We typically have a space after 'if', would you mind adding a space here and in the other instances?